contracts/VaultFactory.vy

```diff
@@ -172,15 +170,44 @@ def protocol_fee_config(vault: address = msg.sender) -> PFConfig:
     @return The protocol fee config for the msg sender.
     """
     # If there is a custom protocol fee set we return it.
-    if self.use_custom_protocol_fee[vault]:
+    config_data: uint256 = self.custom_protocol_fee_data[vault]
+    if self._unpack_custom_flag(config_data):
         # Always use the default fee recipient even with custom fees.
-        return PFConfig({
-            fee_bps: self.custom_protocol_fee[vault],
-            fee_recipient: self.default_protocol_fee_config.fee_recipient
-        })
+        return (
+            self._unpack_protocol_fee(config_data),
+            self._unpack_fee_recipient(self.default_protocol_fee_data)
+        )
     else:
         # Otherwise return the default config.
-        return self.default_protocol_fee_config
+        config_data = self.default_protocol_fee_data
+        return (
+            self._unpack_protocol_fee(config_data), 
+            self._unpack_fee_recipient(config_data)
+        )
+
+@view
+@external
+def use_custom_protocol_fee(vault: address) -> bool:
```

Comment:
would `has_` be more precise than `use_` here? to me it is a clearer and feels less like we are taking some action.

Reply:
while that likely is more accurate, would prefer to keep the ABI's the same to the previous versions that are already deployed with this same selector.